python/instrumentation/openinference-instrumentation-langchain/examples/gemini_tracing.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from langchain_google_genai import ChatGoogleGenerativeAI
+from phoenix.otel import register
+
+# Set Environment Variables
+os.environ["GEMINI_API_KEY"] = ""
```

Comment:
you can just assert this at the top of a file rather than hard coding the assignment